### DISCUSSION TITLE
Welcome to PSPs Discussions!

### DISCUSSION THREAD

**Discussion by Noc2** [w3f/PSPs#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DimaBryuhanov** [comment]
Hi, David. You suggested starting a discussion around a [PSP](https://github.com/w3f/PSPs/pull/17). Please, advise how can we proceed from there. How can we invite expert reviewers?

**Reply by Noc2** [reply]
It seems Xiliang Chen and Jeffrey Burdges already looked at your psp. Feel free to reply and also invite others to take a look at it.
